lib/ExtensionWatcher.js

```diff
@@ -2,40 +2,43 @@ const path = require('path')
 const chokidar = require('chokidar')
 const fsEx = require('fs-extra')
 const async = require('neo-async')
+const EventEmitter = require('events')
 
-const SETTINGS_FOLDER = require('./app/AppSettings').SETTINGS_FOLDER
-const EXTENSIONS_FILE = require('./app/AppSettings').EXTENSIONS_FILE
+const {SETTINGS_FOLDER, EXTENSIONS_FILE} = require('./app/AppSettings')
 
 class ExtensionWatcher {
   constructor () {
     const settingsFolder = process.env.APP_PATH ? path.join(process.env.APP_PATH, SETTINGS_FOLDER) : SETTINGS_FOLDER
     this.configPath = path.join(settingsFolder, EXTENSIONS_FILE)
     this.attachedExtensions = []
+    this.eventEmitter = new EventEmitter()
```

Comment:
ExtensionWatcher itself should emit the events, sorry if that wasn't clear before
rather than having this.eventEmitter, ExtensionWatcher should extend EventEmitter